src/packagedcode/go_mod.py

```diff
@@ -273,9 +277,16 @@ def parse_gosum(location):
         line = line.replace('/go.mod', '')
         parsed_dep = get_dependency(line)
 
+        name = parsed_dep.group('name')
+        namespace = ''
+        if '/' in name:
+            dep_name = name_parser(name)
+            namespace = dep_name.group('namespace')
+            name = dep_name.group('name')
+
```

Comment:
here is my suggestion:
```
ns_name = parsed_dep.group('name')
namespace, _, name = ns_name.rpartition('/')
```
the partition and rpartition functions are your friends most often over regex when splitting strings:
```
>>> 'foo'.rpartition('/')
('', '', 'foo')
>>> 'foo/bar'.rpartition('/')
('foo', '/', 'bar')
>>> 'foo/bar/baz'.rpartition('/')
('foo/bar', '/', 'baz')
```

Reply:
This will be more better than mine. Thanks :)